functions/wcif.js

```diff
@@ -0,0 +1,77 @@
+const fs = require('fs')
+
+const ClearWCIF = {
+  name: 'ClearWCIF',
+  // FIXME: quick helper to make sure we start "from scratch", maybe also
```

Comment:
I would also suggest clearing extensions from both persons and competition as well.